rueidis/tests/gen-test-certs.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Generate some test certificates which are used by the regression test suite:
+#
+#   ./tls/ca.{crt,key}          Self signed CA certificate.
+#   ./tls/redis.{crt,key}       A certificate with no key usage/policy restrictions.
+#   ./tls/client.{crt,key}      A certificate restricted for SSL client usage.
+#   ./tls/server.{crt,key}      A certificate restricted for SSL server usage.
+#   ./tls/redis.dh              DH Params file.
+
+generate_cert() {
```

Comment:
how long are these certificates valid and do we really need to check them into git? 
if so i would like to move them to the .github folder
maybe you can use them for other adapters too

Reply:
These are the same certs from the Redis Storage. We could move them somewhere elsd, for reusing

Reply:
ok can you move this then to the .github

how long are these valid?

Reply:
Certs are now dynamically generated during CI